terraform/vars/terraform-production.tfvars

```diff
@@ -7,6 +7,7 @@ auto_scaling_max_capacity = 15
 
 auth_url         = "https://api.resourcewatch.org"
 forms_api_url             = "https://api.resourcewatch.org/v1"
+alerts_api_url   = "https://data-api.globalforestwatch.org/"
```

Comment:
Need to remove the ending `/` to avoid `//` issues with API calls
`"https://data-api.globalforestwatch.org"`